### PR TITLE
Feature/arena table

### PR DESCRIPTION
**Pull request by emilyseibert**
# PR for #282.

## PR Status
**READY**

## Ready for PR?
- [X] I have updated the CHANGELOG with an entry including a description, a link to my profile, and a link to the issue ticket. This change is filed under an Enhancement or Bug Fix, is shown under the relevant release tag name, and is included in my PR branch.
- [X] I have run `lein run-lint` to check for linting errors.

## Changes Proposed:
- Table Component + Basic Styles
- Arena Table Component + Advanced Styles

## Breaking changes?
- None

## Screenshot of feature:
image

@emilyseibert @oconn @dehli

### CODE REVIEW

File: less/panels/config.less [hunk 1]
```diff
@@ -0,0 +1,8 @@
+@import "../components/header.less";
+
+.config-panel {
+    width: calc(~"100% - 120px");
```
Comment (dehli) [inline less/panels/config.less]: Where does this 120px come from?

File: less/utils/typography.less [hunk 2]
```diff
@@ -17,9 +17,13 @@
 @font-size-18: 18px;
 @font-size-20: 20px;
 @font-size-30: 30px;
- 
+
 @font-size-primary: @font-size-14;
 
 .bold {
```
Comment (dehli) [inline less/utils/typography.less]: Shouldn't we use `<strong>` tags instead of a `.bold` class?

Comment (emilyseibert) [inline less/utils/typography.less]: we should use strong for 'more important text' it's a semantic thing. i think we could argue the header is more important, but its bold is stylistic and isn't standing out amongst other headers

File: src/cljs/wombats_web_client/panels/config.cljs [hunk 3]
```diff
@@ -0,0 +1,15 @@
+(ns wombats-web-client.panels.config
+  (:require [re-frame.core :as re-frame]
+            [wombats-web-client.components.header :refer [header]]
+            [wombats-web-client.components.arena-table
+             :refer [arena-table]]
+            [wombats-web-client.events.arenas :refer [get-arenas]]))
+
+(defn config []
+  (get-arenas)
```
Comment (dehli) [inline src/cljs/wombats_web_client/panels/config.cljs]: Why is this here instead of `component-did-mount`?

File: less/panels/config.less (hunk 1 above)
Comment (emilyseibert) [inline less/panels/config.less]: 60px padding * 2 , will make into a variable
